.github/workflows/ci.yml

```diff
@@ -243,14 +243,17 @@ jobs:
           path: htmlcov
 
   test-typing-extensions:
-    name: test typing-extensions
+    name: test typing-extensions ${{ matrix.typing-extensions-version }} on Python ${{ matrix.python-version }}
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         # test the oldest supported version and main
         typing-extensions-version: ['4.6.1', 'main']
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.8', '3.9', '3.10', '3.11']
+        include:
+          - typing-extensions-version: '4.6.1'
+            python-version: '3.7'
```

Comment:
Why do we even need 4.6.1 here? As far as I can see, we just do simple `make test`.

I think it's enough to run against `main` here. Thus this could be just a list of python versions and no `else` branch in the `install typing-extensions` step.

Reply:
The idea is to test the oldest version we support and the latest version on every python version we support

Reply:
`make test` could very well fail if .e.g we import a symbol that doesn't exist